backend/entityservice/logger_setup.py

```diff
@@ -0,0 +1,90 @@
+import os
+import logging.config
+import structlog
+
+import yaml
+
+
+def setup_logging(
+    default_path='logging.yaml',
+    default_level='INFO',
+    env_key='LOG_CFG'
+):
+    """
+    Setup logging configuration
+    """
+    path = os.path.dirname(os.path.abspath(__file__)) + '/' + default_path
+    value = os.getenv(env_key, None)
+    if value:
+        path = value
+    if os.path.exists(path):
+        with open(path, 'rt') as f:
+            config = yaml.safe_load(f.read())
+        logging.config.dictConfig(config)
+    else:
+        print("YAML file doesn't exist")
```

Comment:
This will be hard to decipher amongst many lines of output.

```suggestion
        print(“Logging configuration YAML file doesn't exist. Falling back to defaults.")
```